safeeyes/glade/settings_dialog.glade

```diff
@@ -627,15 +567,53 @@
                                     <property name="position">1</property>
                                   </packing>
                                 </child>
+                                <child>
+                                  <object class="GtkBox" id="box16_random_order">
+                                    <property name="visible">True</property>
+                                    <property name="can-focus">False</property>
+                                    <property name="spacing">5</property>
+                                    <child>
+                                      <object class="GtkLabel" id="lbl_random_order">
+                                        <property name="visible">True</property>
+                                        <property name="can-focus">False</property>
+                                        <property name="label" translatable="yes">Show breaks in random order</property>
```

Comment:
Please add `Show breaks in random order` to the https://github.com/slgobinath/SafeEyes/blob/master/safeeyes/config/locale/safeeyes.pot#L120 to support translation.

After adding to the pot file we need to run the `update-po.sh` to update all local files but I will run it after merging the PR because it may introduce conflicts with weblate translator.

FYI: To avoid conflicts I usually commit the weblate translations before making any changes in the po files. But pot file can be modified anytime.

Reply:
Will do